### ISSUE TITLE
enterprise id not found in Interactivity request

### ISSUE THREAD

**Issue by mikeyangyo** [slackapi/bolt-python#175]
#### The `slack_bolt` version

slack-bolt==1.0.1
slack-sdk==3.0.0

#### Python runtime version

Python 3.7.6

#### OS info

ProductName:	Mac OS X
ProductVersion:	10.14.6
BuildVersion:	18G103
Darwin Kernel Version 18.7.0: Tue Aug 20 16:57:14 PDT 2019; root:xnu-4903.271.2~2/RELEASE_X86_64

#### Steps to reproduce:

(Share the commands to run, source code, and project settings (e.g., setup.py))

```python
from fastapi import FastAPI, Request
from slack_bolt.async_app import AsyncApp
from slack_bolt.adapter.fastapi.async_handler import AsyncSlackRequestHandler

api = FastAPI()
app = AsyncApp()
app_handler = AsyncSlackRequestHandler(app)

# get_db_connection, get_member_from_action is a middleware to read data from database
# punch_action_handler is a handler to do the punch action
app.action(
    constraints={
        "type": "block_actions",
        "block_id": "punch_block",
        "action_id": "punch_action",
    },
    middleware=[get_db_connection, get_member_from_action],
)(punch_action_handler)

@api.post("/slack/events")
async def endpoint(req: Request):
    return await app_handler.handle(req)
```

### Expected result:

run punch_action_handler function and return response back to channel

#### Traceback log
```
ERROR:uvicorn.error:Exception in ASGI application
Traceback (most recent call last):
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/uvicorn/protocols/http/httptools_impl.py", line 396, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/uvicorn/middleware/proxy_headers.py", line 45, in __call__
    return await self.app(scope, receive, send)
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/fastapi/applications.py", line 179, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/starlette/applications.py", line 111, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc from None
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc from None
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/starlette/routing.py", line 566, in __call__
    await route.handle(scope, receive, send)
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/starlette/routing.py", line 227, in handle
    await self.app(scope, receive, send)
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/starlette/routing.py", line 41, in app
    response = await func(request)
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/fastapi/routing.py", line 183, in app
    dependant=dependant, values=values, is_coroutine=is_coroutine
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/fastapi/routing.py", line 133, in run_endpoint_function
    return await dependant.call(**values)
  File "async_app.py", line 52, in endpoint
    return await app_handler.handle(req)
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/slack_bolt/adapter/starlette/async_handler.py", line 56, in handle
    bolt_resp = await self.app.async_dispatch(to_async_bolt_request(req, body))
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/slack_bolt/adapter/starlette/async_handler.py", line 11, in to_async_bolt_request
    body=body.decode("utf-8"), query=req.query_params, headers=req.headers,
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/slack_bolt/request/async_request.py", line 57, in __init__
    AsyncBoltContext(context if context else {}), self.body
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/slack_bolt/request/async_internals.py", line 15, in build_async_context
    enterprise_id = extract_enterprise_id(payload)
  File "/Users/mikeyang/.virtualenvs/slack-punch-service-venv/lib/python3.7/site-packages/slack_bolt/request/internals.py", line 54, in extract_enterprise_id
    elif "id" in org:
TypeError: argument of type 'NoneType' is not iterable
```
when I trace body got in request, there was a enterprise key in body, but its value is None.
But service is still work at 2020/12/08 20:20 UTC+8

## Requirements

```
aiohttp==3.7.3
async-timeout==3.0.1
attrs==20.3.0
chardet==3.0.4
click==7.1.2
fastapi==0.61.2
gunicorn==20.0.4
h11==0.11.0
httptools==0.1.1
idna==2.10
multidict==5.0.2
pydantic==1.7.2
PyMySQL==0.10.1
python-dotenv==0.15.0
slack-bolt==1.0.1
slack-sdk==3.0.0
starlette==0.13.6
typer==0.3.2
typing-extensions==3.7.4.3
uvicorn==0.12.3
uvloop==0.14.0
XlsxWriter==1.3.7
yarl==1.6.3
```

Please read the [Contributing guidelines](https://github.com/slackapi/bolt-python/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to those rules.

**Comment by seratch** [comment]
@mikeyangyo Thanks for sharing this issue. We've fixed this issue by #169 and the fix is included in version 1.1.1. Could you upgrade the version to the latest? https://github.com/slackapi/bolt-python/releases/tag/v1.1.1

**Comment by mikeyangyo** [comment]
@seratch Thanks for response. It worked after I upgrade slack-bolt package to v1.1.1.
